DuckDuckGo/TabSwitcherViewController.swift

```diff
@@ -114,12 +111,12 @@ class TabSwitcherViewController: UIViewController {
         delegate.tabSwitcher(self, didRemoveTabAt: index)
         collectionView.reloadData()
         refreshTitle()
-    }
-    
-    fileprivate func dismiss() {
         if tabsModel.isEmpty {
             delegate.tabSwitcherDidRequestClearAll(tabSwitcher: self)
         }
+    }
+    
+    fileprivate func dismiss() {
```

Comment:
Looks like you answered this in your other comment... Let me know if you want further clarification.